kolibri/core/assets/src/views/sync/FacilityNameAndSyncStatus/index.vue

```diff
@@ -61,6 +61,7 @@
   import { now } from 'kolibri.utils.serverClock';
   import taskStrings from 'kolibri.coreVue.mixins.commonTaskStrings';
   import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import { PageNames } from '../../../../../../plugins/facility/assets/src/constants';
```

Comment:
We want to avoid importing from a plugin like this. The `core` front-end module cannot have a dependency because if Kolibri was run without the Facility plugin enabled, then the whole app wouldn't build (not that this is a particularly likely situation, but best to avoid the possibility).

It looks like the value you use from this is also being added to the Device plugin's constants too.

If there is some sort of cross-plugin use case for this set of constants then I'd suggest moving them into `core/assets/src/constants.js`, which will then allow them to be imported into any plugin in Kolibri a la `import { SyncPageNames } from 'kolibri.coreVue.vuex.constants';`

If you decide to go that route, then you'll just need to remove them from the Facility and Device plugins and update references to them accordingly.